### PR TITLE
fixed issue with dateutils export

### PR DESCRIPTION
**Pull request by klazbaba**
This PR fixes the issue raised in https://github.com/wix/react-native-calendars/issues/1518

### CODE REVIEW

**Comment by Inbal-Tish** [comment]
@klazbaba Hi. Thanks for the PR! Can you please resolve conflicts?

**Comment by klazbaba** [comment]
> @klazbaba Hi. Thanks for the PR! Can you please resolve conflicts?

The conflict has been resolved.

**Comment by mr-snow2** [comment]
Wasn't the same issue fixed with PR #1553?

**Comment by Inbal-Tish** [comment]
@klazbaba The e2e tests are failing as the app crashes for some reason. Please run `npm run e2e' to check it.

**Comment by klazbaba** [comment]
> @klazbaba The e2e tests are failing as the app crashes for some reason. Please run `npm run e2e' to check it.

This is the part of the test failing. I was unable to find the element with that id. How would you suggest I go about it?
Screen Shot 2021-07-15 at 10 12 07 PM

File: src/calendar/day/index.tsx [hunk 1]
```diff
@@ -102,7 +100,7 @@ export default class Day extends Component<DayProps> {
   render() {
     const {day, marking} = this.props;
     const date = xdateToData(day);
-    const isToday = dateutils.isToday(day);
+    const isToday = dateutils_isToday(day);
```
Comment (Inbal-Tish) [inline src/calendar/day/index.tsx:103]: Change the name of the const instead of alias the function

File: src/expandableCalendar/Context/Presenter.js [hunk 2]
```diff
@@ -44,7 +44,7 @@ class Presenter {
   };
 
   setDate = (props, date, newDate, updateState, updateSource) => {
-    const sameMonth = dateutils.sameMonth(XDate(date), XDate(newDate));
+    const sameMonth = dateutils_sameMonth(XDate(date), XDate(newDate));
```
Comment (Inbal-Tish) [inline src/expandableCalendar/Context/Presenter.js:47]: Change the name of the const instead the name of the function

File: src/expandableCalendar/agendaList.js [hunk 3]
```diff
@@ -109,7 +109,7 @@ class AgendaList extends Component {
 
     if (markToday) {
       const todayString = XDate.locales[XDate.defaultLocale].today || commons.todayString;
-      const isToday = dateutils.isToday(XDate(title));
+      const isToday = dateutils_isToday(XDate(title));
```
Comment (Inbal-Tish) [inline src/expandableCalendar/agendaList.js:112]: Again, better change the name of the const instead of changing the name of the function

File: src/dateutils.js [hunk 4]
```diff
@@ -142,7 +142,7 @@ function getWeekDates(date, firstDay, format) {
   }
 }
 
-module.exports = {
+export {
```
Comment (Inbal-Tish) [inline src/dateutils.js:145]: Why this change? isn't `module.exports` work?

**Comment by Inbal-Tish** [comment]
@klazbaba This id is in Day.index, on the day component testID. Try reverting the change from `module.exports` to `exports` and see if that's the issue

**Comment by klazbaba** [comment]
Tests now pass. I believe this is ready for merge in.

**Review by Inbal-Tish — APPROVED** [review]

**Comment by jte0711** [comment]
I think PR #1553 already fix this issue.

**Comment by klazbaba** [comment]
I am not really sure, cos I still got the issue when I tried pulling from master.

**Comment by klazbaba** [comment]
> I think PR #1553 already fix this issue.

I am not really sure, cos I still got the issue when I tried pulling from master.

**Comment by jte0711** [comment]
> > I think PR #1553 already fix this issue.
> 
> I am not really sure, cos I still got the issue when I tried pulling from master.

I haven't try cloning the repo yet, but I applied fixes from #1553 with patch package and it works well. But I do know this fix hasn't released yet, so it might still be causing error when using the package.
